data/outfits.txt

```diff
@@ -176,8 +177,9 @@ outfit "Scram Drive"
 	thumbnail "outfit/hyperdrive"
 	"mass" 30
 	"outfit space" -30
-	"scram drive" .2
-	"hyperdrive" 1
+	"scram drive" 1
+	deviation .2
```

Comment:
may be "scram deviation", "jump deviation" or something "longer"? -> Because the other attribute is "jump speed" and not just "speed".